r/NEWS.md

```diff
@@ -19,6 +19,11 @@
 
 # arrow 5.0.0.9000
 
+## Breaking changes
+
+* `dplyr::summarize()` on an in-memory Arrow Table or RecordBatch no longer eagerly evaluates. Call `compute()` or `collect()` to evaluate the query.
+* Row order of data from a Dataset query is no longer deterministic. If you need a stable sort order, you should explicitly `arrange()` the query.
```

Comment:
Should we mention `options(arrow.summarise.sort)` here too?

Reply:
Yep, Neal I assume you were leaving the non-breaking changes parts of `NEWS` to be filled in later like we usually do, but it would probably be a good idea to add some bullets describing the changes here and in https://github.com/apache/arrow/pull/10722 before this is merged.

Reply:
Yeah of course, we should fill in NEWS, I was just noting these breaking changes so that I didn't forget.